.github/wiki/Add-new-addon-module.md

```diff
@@ -0,0 +1,14 @@
+To add a new addon or module, follow these steps:
+
+1. Navigate to the `src/templates` folder in the project directory.
+2. Create a new folder depending on the type of the addon/module you want to add:
+   - For addons, create a new folder in the `src/templates/addons` directory.
+   - For the core Terraform files, create a new folder/file in the `src/templates/core` directory.
+
+  Inside the newly created addon/module folder, you can include the code required to generate the templates.
+
+3. Navigate to the `skeleton` folder at the same level as the `src` folder in the project directory.
+4. Add the skeleton folders/files for the addon/module that you are adding inside the corresponding folder in the `skeleton` directory.
+
+> [!NOTE]\
+> Before adding a new addon/module, it is recommended to check the existing ones for reference.
```

Comment:
This formatting is not working as expected for wiki pages (for regular pages it is working fine). On wiki it looks a bit strange, maybe we can use something different? For example:

```suggestion
**NOTE**! Before adding a new addon/module, it is recommended to check the existing ones for reference.
```

![image](https://github.com/nimblehq/infrastructure-templates/assets/475367/f22938c6-149c-4966-b974-d5f03e25dffa)

Reply:
That markdown alert feature has not been supported for Wiki content yet
https://github.com/orgs/community/discussions/16925#discussioncomment-6700896

Using bold text to highlight the note is a nice workaround in the meantime. I'll update it accordingly.

Reply:
Updated in 9bef64b. Here is a screenshot of the change:
<img width="892" alt="Screenshot 2023-08-30 at 13 29 41" src="https://github.com/nimblehq/infrastructure-templates/assets/1896814/106092d2-b021-4c12-9678-d4725391792b">